v03_pipeline/lib/model/dataset_type.py

```diff
@@ -153,10 +153,12 @@ def sample_lookup_table_fields_and_genotype_filter_fns(
             },
         }[self]
 
-    @property
-    def formatting_annotation_fns(self) -> list[Callable[..., hl.Expression]]:
+    def formatting_annotation_fns(
+        self,
+        reference_genome: ReferenceGenome,
+    ) -> list[Callable[..., hl.Expression]]:
         return {
-            DatasetType.SNV_INDEL: [
+            (ReferenceGenome.GRCh38, DatasetType.SNV_INDEL): [
```

Comment:
I opted to just make the formatting function list a function of both `reference_genome` and `dataset_type` instead of just `dataset_type` (with some internal logic conditioning on `reference_genome` [like this](https://github.com/broadinstitute/seqr-loading-pipelines/pull/638/files#diff-bfdea0351ad0697e98a4b8fd1a7a8ace4bf559d5e611b3c69aff92d32f0cfcf7L59)).